tests/test_user.py

```diff
@@ -137,3 +137,179 @@ def test_delete_user_by_id(client: TestClient):
     assert response.json() == {
         "detail": "User not found"
     }
+
+
+def test_update_user_by_id(client: TestClient):
+    user_id = "0df1dacb-67f6-495c-b993-49d06a293765"
+
+    json = {
+        "id": "0df1dacb-67f6-495c-b993-49d06a293765",
+        "email": "update@gmail.com",
+        "username": "update",
+        "address": None,
+        "cellphone_number": None,
+        "role_id": 0,
+    }
```

Comment:
```suggestion
    json = {
        "email": "update@gmail.com",
        "username": "update",
    }
```

Request Body 不能有 `user_id` 和 `role_id`，使用者 ID 和權限是不能被改變的
如果是 `None` 部分可以直接不用寫